CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The change log is available [on GitHub][2].
 0.0.1
```

Comment:
Could you also change version here to `0.1.0`? I guess we will release new version of `o-clock` in some nearest future... And it's better to have major version bumped up.